test/lib/mayaUsd/render/mayaToHydra/testFootPrintNode.py

```diff
@@ -38,12 +38,15 @@ def imageDiffFailThreshold(self):
     def imageDiffFailPercent(self):
         return 0.1
 
+    @classmethod
+    def tearDownClass(cls):
```

Comment:
Same comment as in testFlowViewportAPI.py, should you be using tearDown() here rather than tearDownClass()?